doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+* **Global Planning** - The Global Planning module, of which there could be several specialied implementations, use information from the map(s) to determine a route for the robot to achieve its goal.
+* **Local Planning** - The Local Planning module, of which there could also be several specialized implementations, uses the defined route from the Global Planner and information from robot sensors to handle any required deviations from the route to achieve its goal. 
+* **Robot Interface** - The Robot Interface is an abstraction of the robot platform, providing the means to control the robot and learn about its capabilities. There should be a standard set of messages to control the robot and, perhaps, a standard way to determine a robot's capabilities and attributes. This may require a layer on top of the vendor interface.
+
+![Command Pipeline](./images/Command-Pipeline.png)
+
+## 1.4.2 Support Modules
+
+In addition to the main command chain, there are several supporting modules and subsystems required for a complete system. The implementation of these modules are all outside the scope of the Navigation System. However, the inteface to these components is in scope and the associated requirements are defined. 
+
+* **Mapping** - The Navigation System uses a map to plan the robot's route. The map contains information about known (static) obstacles in the environment. The map is typically created in advance and is available to the Navigation System. The map can also be dynamically updated, to reflect the current state of the environment.
+* **Sensing** - There is a sensing subsystem that can utilize various sensors (and sensor fusion), the goal of which is to develop an understanding of the dynamic environment around the robot. This information is available to the Navigation System, especially the Local Planning module, in directing the robot's motion.  
+* **Localization** - The Navigation System also needs a Localization module to provide the current location of the robot. 
+
+In a complete robot system these modules are utilized by the core navigation modules (the command chain). This document does not specify the architecture for the complete system, but simply names the various comonents and subsystems for the purpose of listing their associated requirements. The information provided by these modules should be available to all of the modules in the command pipeline.
+
+![Command Pipeline with Support](./images/Command-Pipeline-Support.png)
+
+To facilitate error recovery, each module in the chain, if it is unable to carry out its task, its able to propagate error information to its predecessor in the command chain.
+
+## 1.5 Design Goals
+
+The Navigation System designers should strive to meet the following high-level design goals:
+
+* **Modularity** - The Navigation Stack should allow for other implementers to easily replace architectural level components with alternative implementations, in particular, the global and local planners.
```

Comment:
I've been trained to avoid the passive voice. How about, "The Navigation System should allow developers to easily replace components with alternative implementations."